### DISCUSSION TITLE
Changelog and tasks

### DISCUSSION THREAD

**Discussion by EranOfek** [EranOfek/AstroPack#261] [General]
A link to the changelog:
https://docs.google.com/document/d/1CTrcJIZ1IStNLiJPuWravzvfVHVMlOceVVt-iu3dcf0/edit?usp=sharing

A link to the pipeline tasks:
https://docs.google.com/document/d/1y-5KaoadvIEu6ISCLtw2t--r9pWfZhyuDkezuozrHL0/edit?usp=sharing

**Comment by EastEriq** [comment]
Why not part of the repo??

Also what is the branch policy? Is dev1 periodically merged with master?

Il 5 ott 2023 14:33, EranOfek ***@***.***> ha scritto:

A link to the changelog:
https://docs.google.com/document/d/1CTrcJIZ1IStNLiJPuWravzvfVHVMlOceVVt-iu3dcf0/edit?usp=sharing
